lib/nanoc/base/views/item_rep_view.rb

```diff
@@ -27,6 +27,11 @@ def name
       @item_rep.name
     end
 
+    # @return [Boolean]
```

Comment:
I like `modified` (more than `updated` which I use elsewhere) but I’m still hesitant to expose this as part of the public API just yet. I’d prefer to have it just on `PostCompileItemView` for the time being.

(I’ve been bitten by trying to expose too much before, so I’d like to avoid that!)